### PR TITLE
Enhancement/51 backup wasabi

### PR DESCRIPTION
**Pull request by yhan818**
See issue https://github.com/UAL-RE/redata-iac/issues/51

### CODE REVIEW

**Comment by yhan818** [comment]
There is one request not implemented yet (remove old backup).

File: scripts/backup2wasabi.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### This script uses keys to create a new directory in an existing bucket in wasabi using the system's timestamp, and sync files and directories to wasabi.
+ 
+#!/bin/bash
+
+# Create a timestamp for the backup
+#TIMESTAMP=$(date +"%Y-%m-%d_%H-%M-%S")
+TIMESTAMP=$(date +"%Y-%m-%d")
+
+if [ ! -d "./log" ]; then
+    mkdir "./log"
```
Comment (zoidy) [inline scripts/backup2wasabi.sh:13]: Is there a reason this one is a relative path but all the other paths in this file are absolute?

File: scripts/backup2wasabi.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+fi
+
+# Output to a separate log: curation2_backup_log_timestamp.log
+    LOG_FILE="/home/curator/codes/LD_Cool_P/curation/scripts/log/curation2_bk_log_${TIMESTAMP}.log"
```
Comment (zoidy) [inline scripts/backup2wasabi.sh:17]: extra tab is present

File: scripts/backup2wasabi.sh [hunk 3]
```diff
@@ -0,0 +1,87 @@
+#### backup2wasabi.sh
+### Version 1.00.  Date: 2023-10-30
+### Author : Yan Han
+### This script uses keys to create a new directory in an existing bucket in wasabi using the system's timestamp, and sync files and directories to wasabi.
+
```
Comment (zoidy) [inline scripts/backup2wasabi.sh:5]: Please provide examples/documentation of how to run this script.  Also, please provide instructions on how to install the aws dependency

**Review by zoidy — CHANGES_REQUESTED** [review]
The script looks good, just a few simple requests. Also, since the issue this PR references isn't fully addressed, I recommend not merging it until it is. Alternatively, split off the part that remains unaddressed into its own issue

File: scripts/backup2wasabi.sh (hunk 1 above)
Comment (yhan818) [reply scripts/backup2wasabi.sh:13]: right. shall use the absolute path

File: scripts/backup2wasabi.sh (hunk 3 above)
Comment (yhan818) [reply scripts/backup2wasabi.sh:5]: DONE

File: scripts/backup2wasabi.sh (hunk 1 above)
Comment (yhan818) [reply scripts/backup2wasabi.sh:19]: I created a new issue for removing old copies.

Comment (yhan818) [reply scripts/backup2wasabi.sh]: Updated the log. see the latest commit.

**Review by zoidy — APPROVED** [review]

**Pull request merged**